### ISSUE TITLE
Okta Chrome Tab closes on background

### ISSUE THREAD

**Issue by RichiCoder1** [okta/okta-oidc-android#197]
## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior

When calling `signIn`, it will navigate to the custom tab. However, backgrounding this tab will cause it to close and loose context. This breaks password manager flows where it needs to temporarily navigate to a different fragment.

## Expected behavior
Backgrounding the custom chrome tab will keep it open and available.

## Environment

- Package Version: 1.4.4
- React Native Version: 0.62.2
- OS: Android v11
- Node version (`node -v`): v12.8.1
- Other:

**Comment by RichiCoder1** [comment]
Glad to know I wasn't crazy :). Looking forward to the fix, this is def critical to our Password Manager-using users.

**Comment by JayNewstrom** [comment]
@RichiCoder1 while we're searching for the ideal solution, you can go back to a previous version to allow password manager users to successfully complete the flow.

You can see the diff from older version with a link like this https://github.com/okta/okta-oidc-android/compare/1.0.15...1.0.16

I would suggest going back to `1.0.15`, but you could also choose `1.0.14`.

You can force this by adding a block similar to this to the Android build.gradle.

```
configurations.all {
    resolutionStrategy {
        force 'com.okta.android:oidc-androidx:+', 'com.okta.android:oidc-androidx:1.0.15'
    }
}
```

**Comment by JayNewstrom** [comment]
@RichiCoder1 we fixed this in #202

**Comment by JayNewstrom** [comment]
This should be fixed in `1.0.17`.
